odoo/addons/base/ir/ir_translation.py

```diff
@@ -469,7 +469,7 @@ def _sync_terms_translations(self, field, records):
                 if trans.src == trans.value:
                     discarded += trans
                 elif trans.src not in terms:
-                    matches = get_close_matches(trans.src, terms, 1, 0.9)
+                    matches = get_close_matches(trans.src, terms, 1, 1)
```

Comment:
This screws up a functionality: it allows to not lose translations when you fix a typo.
This is clearly wrong.